addons/FigureDrawing/src/presenter.js

```diff
@@ -1214,6 +1226,48 @@ function AddonFigureDrawing_create(){
             if (presenter.coloring) presenter.redrawCanvas(false);
         }
     }
+
+    presenter.getActivitiesCount = function () {
+        if (!presenter.activity) return 0;
+        return presenter.AnswerLines.length;
+    }
+
+    presenter.isEnabledInGSAMode = function() {return true;};
+
+    presenter.gradualShowAnswers = function (item) {
+        if (presenter.activity) {
+            var i, x, y, point1, point2;
+            if (presenter.isShowAnswersActive) presenter.hideAnswers();
+            presenter.isShowAnswersActive = true;
+            presenter.isGradualShowAnswersActive = true;
+            presenter.setWorkMode();
+            var Lines = presenter.$view.find('.line').not('.nonremovable');
```

Comment:
Widzę, że ta metoda została stworzona na podstawie showAnswers. Z tego powodu rozumiem powód braku zmian nazw. Jak będzie potrzeba zmiany w obu miejscach to prosiłbym o zmianę nazwy na `$lines`, ze względu na:
1. Wynikiem jest obiekt jQuerry
2. Zmienna nie powinna zaczynać się z dużej litery